demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, query: str, insecure: bool = False, debug: str = None, debug_
         if self.debug:
             self.query += ' debug-mode="true"'
 
+    def set_insecure(self, insecure):
```

Comment:
why not pass it in init?

Reply:
since it is a demisto_client object, so it will require more imports and internals changes in the `Runner` object that will occur in `common.py`. I though adding a set function is a cleaner approach. but - whatever you'll say.